jordinner/src/pages/restaurants/signin.jsx

```diff
@@ -1,3 +1,78 @@
-export default function Signin() {
-    return <p>Something</p>
-}
\ No newline at end of file
+import React, { useState } from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+export default function SignIn() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [successMessage, setSuccessMessage] = useState('');
+  const router = useRouter();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    if (!email || !password) {
+      setSuccessMessage('Please fill in all fields.');
+      return;
+    }
+
+    if (!email.includes('@') || !email.includes('.')) {
+      setSuccessMessage('Please enter a valid email address.');
+      return;
+    }
+
+    if (password.length < 6) {
+      setSuccessMessage('Password must be at least 6 characters long.');
+      return;
+    }
```

Comment:
According to our schema, the password is 8 character's long, but ultimately, there's no need to give validation errors when signing in. We just need to return an error message stating that there's an issue with either the email or password, without specifying